x-pack/plugins/siem/public/components/page/network/users_table/columns.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { StaticIndexPattern } from 'ui/index_patterns';
+
+import { FlowTarget, UsersItem } from '../../../../graphql/types';
+import { networkModel } from '../../../../store';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { getEmptyTagValue } from '../../../empty_value';
+import { Columns } from '../../../load_more_table';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+import * as i18n from './translations';
+import { getRowItemDraggables } from '../../../tables/helpers';
+
+export const getUsersColumns = (
+  indexPattern: StaticIndexPattern,
+  ip: string,
+  flowTarget: FlowTarget,
+  type: networkModel.NetworkType,
+  tableId: string
+): [
+  Columns<UsersItem['name']>,
+  Columns<UsersItem['id']>,
+  Columns<UsersItem['groupName']>,
+  Columns<UsersItem['groupId']>,
+  Columns<UsersItem['count']>
+] => [
+  {
+    field: `node.user.name`,
+    name: i18n.USER_NAME,
+    truncateText: false,
+    hideForMobile: false,
+    sortable: true,
+    render: userName => {
+      const userNameAttr = 'user.name';
+      if (userName != null) {
+        const id = escapeDataProviderId(`${tableId}-table-${flowTarget}-user-${userName}`);
+        return (
+          <DraggableWrapper
+            key={id}
+            dataProvider={{
+              and: [],
+              enabled: true,
+              id,
+              name: userName,
+              excluded: false,
+              kqlQuery: '',
+              queryMatch: { field: userNameAttr, value: userName },
```

Comment:
If this is the only place the variable is used, I would just put the string directly in the field.